client/src/views/Login.tsx

```diff
@@ -1,8 +1,67 @@
+import React, { useState } from "react";
+import { useAuthStore } from "../store/authStore";
+import InputField from "../components/InputField";
+import FormError from "../components/FormError";
+
 export default function Login() {
+  const { login } = useAuthStore();
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
+  const [error, setError] = useState<string | null>(null);
+  const [success, setSuccess] = useState<string | null>(null);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setError(null);
+    setSuccess(null);
+    try {
+      await login(formData.email, formData.password);
+      setSuccess("Login successful!");
+    } catch (err) {
```

Comment:
that's never used and eslint throw this error.

if you have an eslint error you can't build the frontend.